layouts/css/layout/_sticky-footer.styl

```diff
@@ -9,6 +9,8 @@ body
     flex-direction column
     min-height 100vh
     margin 0
+    overflow-y scroll; /* has to be scroll, not auto */
+    -webkit-overflow-scrolling touch
```

Comment:
What's the standard way to turning on the momentum scrolling?    I really do enjoy browsing through the docs on my phone and it's such a pain on my thumb when I have to use it so much to find my spot again (this is the only way I know how :/).    

> On Dec 9, 2015, at 7:29 PM, Steve Mao <notifications@github.com> wrote:
> 
> In layouts/css/layout/_sticky-footer.styl:
> 
> > @@ -9,6 +9,8 @@ body
> >      flex-direction column
> >      min-height 100vh
> >      margin 0
> > +    overflow-y scroll; /* has to be scroll, not auto */
> > +    -webkit-overflow-scrolling touch
> I think its already configured.
> 
> overflow-scrolling touch should just work but its not really standard.
> 
> —
> Reply to this email directly or view it on GitHub.
>